clusterloader2/pkg/measurement/common/waitforpods.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/perf-tests/clusterloader2/pkg/measurement"
+	"k8s.io/perf-tests/clusterloader2/pkg/util"
+)
+
+func init() {
+	measurement.Register("WaitForPods", createWaitForPodMeasurment)
+}
+
+func createWaitForPodMeasurment() measurement.Measurement {
+	return &waitForPodMeasurment{}
+}
+
+type waitForPodMeasurment struct{}
+
+// Execute waits until desired number of pods is available or until timeout happens.
+// Pods can be specified by field and/or label selectors.
+// If namespace is not passed by parameter, all-namespace scope is assumed.
+func (wfpm *waitForPodMeasurment) Execute(config *measurement.MeasurementConfig) error {
```

Comment:
I suggest wfpm -> w

Reply:
replace with `_` - structure is not used.